projects/storefrontlib/src/cms-components/navigation/category-navigation/category-navigation.module.ts

```diff
@@ -11,6 +11,9 @@ import { CategoryNavigationComponent } from './category-navigation.component';
       cmsComponents: {
         CategoryNavigationComponent: {
           component: CategoryNavigationComponent,
+          data: {
```

Comment:
You can use another `provideDefaultConfig()` to import this configuration and remove it from component data.

`in separate default config file for component`
```
export const defaultCategoryNavigationConfig: NavigationUiConfig = {
	resetMenuOnClose: true
}
```

`in component module`
```
...
providers: [
	provideDefaultConfig(defaultCategoryNavigationConfig),
	...
```